test/kubernetes/e2e/defaults/testdata/nginx_pod.yaml

```diff
@@ -38,67 +38,27 @@ data:
-    AAOCAQ8AMIIBCgKCAQEAw6lxj3IX6kBNKWF0LEQiONJN81vbKNDEVpE+w5zwaA1K
-    zAMSfKxkhdQMPtM+MS64CPkDUZFxdYbUgKygl23uWcuIPWHnD7aqICm+ujMLUMzw
-    RFXablUCZiO9sFfZegkdLwvecmtnvNjVL5s8jk3HjV8Jetu/tE17HMvP4cMdfs/r
-    zdYRxoI2tWyYDWUW1XfD6eTxDykWwfLMdJ6UX0ksZSlQ098OheMMA6E+cxH0JMoe
-    +PLyD4nuAYW8c6tOFTXJjqHUaJzSRlYFg3OG0WRWKcjP9ufeLsPWjWza5M6WSGEj
-    hiPP2bSxMCfkY3DFSO3K71MrYf5xsP4L70YmD2oUowIDAQABo1kwVzAUBgNVHREE
-    DTALgglsb2NhbGhvc3QwCwYDVR0PBAQDAgeAMBMGA1UdJQQMMAoGCCsGAQUFBwMB
-    MB0GA1UdDgQWBBSC36i8yBxdER9rU3KJvR/Dtop9XTANBgkqhkiG9w0BAQsFAAOC
-    AQEAXG1LQfCVJZ3R2rXHZirUHkSgXCPMglUv9dN25XMvGiOwVGX8g9QUv+WuMeoS
-    VK98rLnej7EOLZLb+02lXKqAT8G6eDqXlZONfFCTnS6BWc0+5o2fvnniuJhzxGq0
-    qolf2q4P4JNJk7TRlulaLdIxSOMyJukRne4kRcbkz3SaVE+eGAm6IURSEE1x1AXU
-    BaCZpm5MWgiJtOJulM7/9Nw8SpTir3nKNTcI3Q0M2XGvhWylN9N17AkANrBrNBme
-    LDyhBvUlZrbnOxfblBxzB8jocGxCLDLLtNNlfuEPquy8J263LVIh+Totibxhg4l/
-    MaeHQu7bsnjxU6pWF3x/QsZYzQ==
-    -----END CERTIFICATE-----
-  localhost.key: |
-    -----BEGIN PRIVATE KEY-----
-    MIIEvAIBADANBgkqhkiG9w0BAQEFAASCBKYwggSiAgEAAoIBAQDDqXGPchfqQE0p
-    YXQsRCI40k3zW9so0MRWkT7DnPBoDUrMAxJ8rGSF1Aw+0z4xLrgI+QNRkXF1htSA
-    rKCXbe5Zy4g9YecPtqogKb66MwtQzPBEVdpuVQJmI72wV9l6CR0vC95ya2e82NUv
-    mzyOTceNXwl627+0TXscy8/hwx1+z+vN1hHGgja1bJgNZRbVd8Pp5PEPKRbB8sx0
-    npRfSSxlKVDT3w6F4wwDoT5zEfQkyh748vIPie4Bhbxzq04VNcmOodRonNJGVgWD
-    c4bRZFYpyM/2594uw9aNbNrkzpZIYSOGI8/ZtLEwJ+RjcMVI7crvUyth/nGw/gvv
-    RiYPahSjAgMBAAECggEAPgk8WznWgUwv6t3eQqa7nv52/qDyJMfEaJoWp1jcFHGa
-    dILI2sSh/piT5Vt+zYM9kIK7XaJaseO0/rM9G6YcO1Y/9Q5Uf3PwDtCNi2XhwZWo
-    3FHINtE3OIE+hGnmJz46hY8i0W5ibsrlFPoIQipBCf7G97Ay4Qzr6t5oI/GEKY9e
-    9b3/CJ12cvpjrrEnvvgcQv9XJ83dYVpvBLL8Hm9WjwFsmV1erN3JXB+yHe8b3MHK
-    BGA5rM+bpn+LxV3b7ycnfLXjPVkKY9KXBQCyZDzsGaCfzmQ7dJqIJAdtjR320A3A
-    M3RU/YCfV7lVxuCuPK59N3xhJJFDnfrikkKLWuGYmQKBgQDX2/iMnvPKjgcgYyui
-    Gol7h8VqRpQycXHUS89zCeIhZPr6i5ZjzltblOFhsSE26UP6S9QXG8Hw7v6dEKDx
-    9zVgNK75e1vzr8Gm5Ld+eqo22PQR1RHeQkwJXgXL+3GfHxFes6TG2h9MR3TNBGJU
-    EOayfzeNT6l3JSUV/ae913ognQKBgQDoC/jiytQz8auV4p2c/aiMRziL+xgJkV8f
-    E7q0pw4BZirRf6N6YdNU/mOhgJd1LHkcgYdnPbxHXIhNUUE3uS9Uvc+exujh4R3a
-    FcmUcbEjBG/yRVFDtgmszQtA+U+1upu4JG5MhuzLizGZvYWD0VrpGrKIBOwxlvEW
-    zMFp0JvmPwKBgCBuBdtqjgniaKOvAoEqJ3mNnlUnIWCqtoVElngcBgMqXqKBkiiQ
-    eh06MtoweGL9jJ7wAX8vRmXiIhKKywNPNo+rmpYUuG3V++wM9Jxl5Wi0E4cSUcro
-    fu/xVkGdFybmzf9CUgEmCAm3uo6KmBM1LtOmVTw/uaASzo2NPERDOS/pAoGAWpwq
-    MK0RFcN9xAZ8k0v9n+FDtG11Im9QnHsAwgAlmOhDOhFETcqbUioPz4W+HrQiCr6N
-    mAPkXF1GoCJlfBPk5otD4nU7hNB57qnpT/zhNZJLAGiO5gjUWFSs208/D/BxVANt
-    ypY5KvYMhUMbOrDqdfHF2xVJAcg2FjgYInCiH9MCgYAYGrAdT7gDLjjNtSuoFur7
-    sMtACf59rw24my1UoWCjVOxE1MVWRtCYDV5Qi+7MQ4xNejugKKKNiRwrBa6Z6bCm
-    JnPicvcV8IaWwKQqHBjDQn47DVQ+xXNHara6Alv/kjpyJztXsh12S7Hn2EJY91//
-    hcbfh0ySbbsqheBkTXMrvQ==
-    -----END PRIVATE KEY-----
+---
+apiVersion: v1
+kind: Secret
+type: kubernetes.io/tls
+metadata:
+  name: nginx-tls
+  namespace: nginx
+data:
```

Comment:
not sure what is the best practice here. if possible storing the certs (even testing ones) as gh secrets can be a better way.